docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -0,0 +1,73 @@
+6. De ZGW APIs worden aangetoond middels een referentie implementatie om zo 
+   leveranciers te helpen met praktische keuzes.
+7. De ZGW APIs zijn strakker gedefinieerd dan de ZDS waardoor er minder 
+   verschillen tussen implementaties zal ontstaan.
+
+
+## Vergelijking
+
+### Operaties
+
+ZDS 1.2 | ZGW 1.0-alpha \*
+--- | ---
+Geef Zaakstatus (`geefZaakstatus_ZakLv01`)                              | Zaken API `GET /statussen?zaak=<ID>`  
+Geef Zaakdetails (`geefZaakdetails_ZakLv01`)                            | Zaken API `GET /zaken/<ID>` 
+Actualiseer Zaakstatus (`actualiseerZaakstatus_ZakLk01`)                | Zaken API `POST /statussen/`
+Creëer Zaak (`creeerZaak_ZakLk01`)                                      | Zaken API `POST /zaken`
+Update Zaak (`updateZaak_ZakLk01`)                                      | Zaken API `PUT /zaken/<ID>`
+Genereer Zaakidentificatie (`genereerZaakIdentificatie_Di02`)           | *niet geïmplementeerd, in overweging* <sup>1</sup>
+Geef lijst Zaakdocumenten (`geefLijstZaakdocumenten_ZakLv01`)           | Documenten API `GET /objectinformatieobjecten`
+Geef Zaakdocument lezen (`geefZaakdocumentLezen_EdcLv01`)               | Documenten API `GET /enkelvoudiginformatieobjecten/<ID>`
+Geef Zaakdocument bewerken (`geefZaakdocumentbewerken_Di02`)            | *niet geïmplementeerd, komt binnenkort* <sup>2</sup>
+Voeg Zaakdocument toe (`voegZaakdocumentToe_EdcLk01`)                   | Documenten API `POST /enkelvoudiginformatieobjecten`
+Maak Zaakdocument (`maakZaakdocument_EdcLk01`)                          | *niet geïmplementeerd, in overweging* <sup>3</sup>
+Update Zaakdocument (`updateZaakdocument_Di02`)                         | Documenten API `POST /enkelvoudiginformatieobjecten`    
+Genereer Documentidentificatie (`genereerDocumentIdentificatie_Di02`)   | *niet geïmplementeerd, in overweging* <sup>1</sup>
+Cancel CheckOut (`cancelCheckout_Di02`)                                 | *niet geïmplementeerd, komt binnenkort* <sup>2</sup>
+CMIS-integratieservice (*geen operatie*)                                | *n.v.t.*
+Koppel Zaakdocument aan Zaak (*bestaat niet*)                           | Documenten API `POST /objectinformatieobjecten`
+Ontkoppel Zaakdocument (`ontkoppelZaakdocument_Di02`)                   | *niet geïmplementeerd, komt binnenkort* <sup>4</sup>
+Voeg besluit toe (`voegBesluitToe_Di01`)                                | Besluiten API `POST /besluiten`
+Update Besluit (`updateBesluit_BslLk01`)                                | Besluiten API `PUT /besluiten`
+Genereer Besluit Identificatie (`genereerBesluitIdentificatie_Di02`)    | *niet geïmplementeerd, in overweging* <sup>1</sup>
+Geef Besluitdetails (`geefBesluitDetails_BslLv01`)                      | Besluiten API `GET /besluiten/<ID>`
+Geef lijst Besluiten (`geefLijstBesluiten_ZakLv01`)                     | Besluiten API `GET /besluiten`
+Overdragen te behandelen Zaak (`overdragenZaak_Di01`)                   | *n.v.t.*
+
+
```

Comment:
Misschien hier ook Regie- Zaakservices erbij halen en vergelijken. Daar zit bijv. overdragenZaak en notificeerOverZaak.

Reply:
Ik heb geen kennis van dit koppelvlak. Moeten we dat in dit stadium wel betrekken? Dat schept verwachtingen en ik kan niet inschatten of we die waar kunnen maken.

Reply:
Eens. Ik heb het ook al bij Ivo genoemd m.b.t. notificaties.